aws/resource_aws_db_instance.go

```diff
@@ -407,11 +437,11 @@ func resourceAwsDbInstanceCreate(d *schema.ResourceData, meta interface{}) error
 			opts.DBSubnetGroupName = aws.String(attr.(string))
 		}
 
+		// TODO: Only allow this param if the master is not encrypted or
```

Comment:
Is this still an outstanding concern?

Reply:
@jen20 I wrote that based on the ambiguity in the AWS documentation I describe above.  When creating a replica, they seem to have two situations where they allow an encryption key to be specified for a replica.  When creating an encrypted replica in the same region as the master, you cannot specify the key.

I had thought that maybe we should show an error if a key was specified for a same-region replica.

However, maybe it is best to let the AWS API throw an error if this param is used when it is not allowed.

Unless you have advice on which way to go, I can take those lines out.  The PR is working as-is for us.